automatic/PDFCreator/update.ps1

```diff
@@ -0,0 +1,24 @@
+import-module au
+
+$url = 'http://download.pdfforge.org/download/pdfcreator/PDFCreator-stable?download'
+
+function global:au_SearchReplace {
+    @{
+        'tools\ChocolateyInstall.ps1' = @{
+            "(^[$]checksum\s*=\s*)('.*')"   = "`$1'$($Latest.Checksum32)'"
+        }
+     }
+}
+
+function global:au_GetLatest {
+    Write-Host $url
+
+    $temp_file = $env:TEMP + '\PDFCreator.exe'
+    Invoke-WebRequest $url -OutFile $temp_file
+
+    $version = (Get-Command $temp_file).Version
```

Comment:
OK, I can look at the RSS feed for the first title that contains "released" and extract the version. It might not be perfect, but it's probably good enough.